src/org/zaproxy/zap/extension/api/OptionsParamApi.java

```diff
@@ -48,16 +50,34 @@ public OptionsParamApi() {
     @Override
     protected void parse() {
         
-	    enabled = getConfig().getBoolean(ENABLED, true);
-	    secureOnly = getConfig().getBoolean(SECURE_ONLY, false);
-		disableKey = getConfig().getBoolean(DISABLE_KEY, false);
-		incErrorDetails = getConfig().getBoolean(INC_ERROR_DETAILS, false);
-		autofillKey = getConfig().getBoolean(AUTOFILL_KEY, false);
-		enableJSONP = getConfig().getBoolean(ENABLE_JSONP, false);
-	    key = getConfig().getString(API_KEY, "");
-	    //postActions = getConfig().getBoolean(POST_ACTIONS, false);
+	    enabled = getBooleanFromConfig(ENABLED, true);
+	    secureOnly = getBooleanFromConfig(SECURE_ONLY, false);
+		disableKey = getBooleanFromConfig(DISABLE_KEY, false);
+		incErrorDetails = getBooleanFromConfig(INC_ERROR_DETAILS, false);
+		autofillKey = getBooleanFromConfig(AUTOFILL_KEY, false);
+		enableJSONP = getBooleanFromConfig(ENABLE_JSONP, false);

```

Comment:
No reason, it's just that I didn't need to change the indentation of the statements. I'll change that (worth noting that the first two statements are the ones with "wrong" indentation, using mixed chars).